### PR TITLE
added coreutils dependency + upgraded dependencies

### PR DESCRIPTION
**Pull request by superherointj**

### CODE REVIEW

File: examples/api-gateway/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN echo ' \
 {\
   "name": "package-base", \
   "dependencies": { \
-    "ocaml": "~4.7.1000", \
+    "ocaml": "*", \
```
Comment (anmonteiro) [inline examples/api-gateway/Dockerfile]: We want this to be the latest version that we support, so it should be synced with whatever is in `esy.json` (`4.9.0` at the moment)

Comment (superherointj) [reply examples/api-gateway/Dockerfile]: Done.

**Comment by superherointj** [comment]
Anything else?

**Comment by anmonteiro** [comment]
Thanks!

**Pull request merged**